src/sqlancer/sqlite3/oracle/SQLite3NoRECOracle.java

```diff
@@ -90,11 +116,12 @@ public void check() throws SQLException {
             throw new IgnoreMeException();
         }
         if (optimizedCount != unoptimizedCount) {
```

Comment:
I think we could simplify the code by adding `reproducer = new SQLite3NoRECReproducer(optimizedQuery, unoptimizedQuery, Arrayas.asList(optimizedQueryString, unoptimizedQueryString));` above the if check and removing the other parts.